### ISSUE TITLE
apollo datasets

### ISSUE THREAD

**Issue by JingyuLi-code** [cardwing/Codes-for-IntRA-KD#7]
thanks for your sharing codes ! in your paper, apollo dataset Validation is 10000 images, but your code vallist only have 1000 images？ how about this problem?
![image](https://user-images.githubusercontent.com/45377489/85936636-2f83f500-b92f-11ea-8858-f44e2f3cc359.png)

**Comment by cardwing** [comment]
Sorry about the mistake. You can just randomly split one subset (around 10 K images) from the original training set as the validation set. The performance in our paper is reported on the testing set.

**Comment by JingyuLi-code** [comment]
Thanks for your replay! Thanks again！or you can update the val list. goode luck!
